typescript/nomad-monitor/src/googleSheets.ts

```diff
@@ -31,8 +31,10 @@ async function uploadDeployedTokens(
   credentialsFile: string = './credentials.json',
 ) {
   const credentials = JSON.parse(fs.readFileSync(credentialsFile, 'utf8'));
+  // Production Spreadsheet ID: 1RooLLPTtvFaiiiuJH381fcRRgGTajv7w973xUThy5-4
+  // Development Spreadsheet ID: 1AApo2bkGdCRN2w6CJYRF_rIkEnrIt1Ab4C_YZ_FnSn8
```

Comment:
@yourbuddyconner should this use a config value to switch between dev and production?

Reply:
Yes absolutely, I just didn't have the minutes to refactor it last night when I was working on it. Ideally this would switch based on the configured environment.